daily-challenges/2024-09/17/ththao27.spec.js

```diff
@@ -0,0 +1,54 @@
+import { test, expect } from "@playwright/test"
+
+//Javascript
+function isPrime(n) {
+    if (n <= 1) {
+        console.log("Số này không phải là số nguyên tố");
+        return false;
+    }
+
+    for (let i = 2; i <= Math.sqrt(n); i++) {
+        if (n % i === 0) {
```

Comment:
Kết quả sai với số 2: số 2 là số nguyên tố

Reply:
à cái này em dùng điều kiện là i < căn bậc 2 của n nên nếu n bằng 2 thì nó sẽ không nhảy vô vòng for này đâu ạ.
Này e đã test với 2 rồi vẫn trả ra kết quả 2 là số nguyên tố